src/main/java/randoop/sequence/MappedSequences.java

```diff
@@ -17,10 +18,15 @@
  */
 public class MappedSequences<K> {
 
+  /** Maps a key to a collection of sequences. */
   private Map<K, SequenceCollection> map;
 
+  /** Maps a key to a map of sequences to frequency. */
```

Comment:
This doesn't feel like it belongs in MappedSequences.  It is not always needed.  Should it be in a subclass?  Or should there be a different class to hold this information?